### DISCUSSION TITLE
azure openai返回的数据结构，我想要获取这些信息有吗？

### DISCUSSION THREAD

**Discussion by 5478-lfy** [lobehub/lobehub#1193] [General Question]
<OpenAIObject chat.completion id=chatcmpl-8mZRrunXlLdTASoztjnCdJowps9ZG at 0x7fc1a2106a90> JSON: {
  "id": "chatcmpl-8mZRrunXlLdTASoztjnCdJowps9ZG",
  "object": "chat.completion",
  "created": 1706586699,
  "model": "gpt-4",
  "prompt_filter_results": [
    {
      "prompt_index": 0,
      "content_filter_results": {
        "hate": {
          "filtered": false,
          "severity": "safe"
        },
        "self_harm": {
          "filtered": false,
          "severity": "safe"
        },
        "sexual": {
          "filtered": false,
          "severity": "safe"
        },
        "violence": {
          "filtered": false,
          "severity": "safe"
        }
      }
    }
  ],
  "choices": [
    {
      "finish_reason": "stop",
      "index": 0,
      "message": {
        "role": "assistant",
        "content": "- \u6807\u9898\uff1a\u6e05\u534e\u5927\u5b66\u51fa\u54c1\uff01AI\u667a\u80fd\u4f53GitAgent\uff01\n- \u6982\u8981\uff1a\u6e05\u534e\u5927\u5b66\u4e0e\u9762\u58c1\u667a\u80fd\u5408\u4f5c\u5f00\u53d1\u4e86GitAgent\uff0c\u4e00\u79cd\u80fd\u591f\u81ea\u52a8\u5728GitHub\u4e0a\u641c\u7d22\u3001\u914d\u7f6e\u3001\u5e94\u7528\u548c\u50a8\u5b58\u5de5\u5177\u7684AI\u667a\u80fd\u4f53\uff0c\u65e8\u5728\u6269\u5c55\u5927\u6a21\u578b\u667a\u80fd\u4f53\u7684\u80fd\u529b\uff0c\u4ee5\u66f4\u597d\u5730\u5b8c\u6210\u590d\u6742\u4efb\u52a1\u3002\n- \u539f\u6587\u94fe\u63a5\uff1ahttp://mp.weixin.qq.com/s?__biz=MzkyNTUzNjg2OA==&mid=2247487879&idx=1&sn=d8f6db0f020721108e47982803f263f3&chksm=c1c4568cf6b3df9ae5899140effa1f5b9053e28f2f454a448c7dd78ecda43c4eac83624c14a3#rd"
      },
      "content_filter_results": {
        "hate": {
          "filtered": false,
          "severity": "safe"
        },
        "self_harm": {
          "filtered": false,
          "severity": "safe"
        },
        "sexual": {
          "filtered": false,
          "severity": "safe"
        },
        "violence": {
          "filtered": false,
          "severity": "safe"
        }
      }
    }
  ],
  "usage": {
    "prompt_tokens": 1410,
    "completion_tokens": 215,
    "total_tokens": 1625
  },
  "system_fingerprint": "fp_6d044fb900"
}

**Comment by arvinxx** [comment]
为啥要，在哪要
